pom.xml

```diff
@@ -231,6 +276,231 @@
 
   <profiles>
     <profile>
+      <id>checkstyle</id>
```

Comment:
I am not sure we need separate profiles for checkstyle + findbugs.. They can be integrated into default build...